src/views/Error.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="container">
+    <!-- 404 error starts -->
+    <div v-if="isPageNotFound">
+      <h1>404</h1>
+      <p>You have visited a wrong link</p>
+      <p>आप गलत लिंक पर आ गए हैं</p>
```

Comment:
Maybe add a link to the home page here.. or may be not. since we don't yet want them to visit the list